cached-mamba/action.yml

```diff
@@ -31,13 +31,15 @@ runs:
       channel-priority: strict
       activate-environment: my-env
       use-mamba: true
+  - name: Set cache date and number
```

Comment:
```suggestion
  - name: Set cache date
```

I guess it's just the date

Reply:
Ah yes, you removed the number at some point.